### PR TITLE
Added client tiles to Homepage

### PR DESCRIPTION
**Pull request by cdccollins**
Added client tiles to the 'Who we work with' section of the homepage and rearranged some, according to trello
Deleted GDS logos from assets/images/homepage, as Gov UK replaces them

### CODE REVIEW

**Comment by ubxd** [comment]
Deploy preview for *unboxed-co-production* ready!

Built with commit 06c3b0b1e48e2820e5cf6a2336fbd1eeaa2bc7e8

https://deploy-preview-468--unboxed-co-production.netlify.com

**Review by neilvanbeinum — APPROVED** [review]

File: source/index.html.erb [hunk 1]
```diff
@@ -38,28 +38,36 @@ products:
...
-      image: "/assets/images/homepage/logo-funding-circle.gif"
+    - name: "Southwark CCG"
+      image: "/assets/images/homepage/logo-southwark-ccg.jpg"
     - name: "Guy's and St Thomas' NHS Foundation Trust"
       image: "/assets/images/homepage/logo-gstt.gif"
     - name: "Apps for Good"
       image: "/assets/images/homepage/logo-apps-for-good.gif"
-    - name: "Government Digital Service"
-      image: "/assets/images/homepage/logo-gds.jpg"
+    - name: "Government UK"
```
Comment (h-lame) [inline source/index.html.erb:55]: Worth checking with @GramMcCram I think, but "Government UK" doesn't sound right to me.  We'd talk about "the UK Government" or "GOV.UK" but not "Government UK".

Comment (h-lame) [reply source/index.html.erb:55]: Oh! I was too slow for this PR as, but it's still worth checking I reckon!

Comment (GraemeMcCubbin) [reply source/index.html.erb:55]: I agree with @h-lame - 'GOV.UK' would be a more suitable name for this tile.

**Pull request merged**